editors/code/package.json

```diff
@@ -949,6 +954,11 @@
                         "type": "string"
                     }
                 },
+                "rust-analyzer.enableBackspaceListener": {
```

Comment:
You'll have to add this before the `                "$generated-start": {},` marker as everything after is auto generated